README.md

```diff
@@ -138,9 +138,9 @@ _To execute the following commands, open a CMD, Bash, or Powershell window by na
     # On Linux of Mac:
    ./run.sh start
    # On Windows:
-   ./run.bat start
+   .\run.bat
```

Comment:
On Windows, . is in searching path. So \. is not necessary at all. It's just `run.bat`.